samples/apps/local-inference-server/utils/CompletionGenerator.py

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
should add file headers `# Copyright (c) Microsoft. All rights reserved.`

Reply:
Good catch, added!